### PR TITLE
Safeguard for Plate Solution

### PR DESCRIPTION
**Pull request by tamimfatahi**
Completes a portion of issue #108

### CODE REVIEW

File: exotic.py [hunk 1]
```diff
@@ -542,11 +542,22 @@ def check_wcs(fits_file, saveDirectory):
 def plate_solution(fits_file, saveDirectory):
     default_url = 'http://nova.astrometry.net/api/'
 
-    # Login to Exoplanet Watch's profile w/ API key
-    r = requests.post(default_url + 'login', data={'request-json': json.dumps({"apikey": "vfsyxlmdxfryhprq"})})
+    # Login to Exoplanet Watch's profile w/ API key. If session fails, allow 5 attempts of
```
Comment (rzellem) [inline exotic.py:545]: Isn't it only trying 4 times?

Comment (rzellem) [reply exotic.py:545]: `for i in range(4)` -> 0, 1, 2, 3

Comment (tamimfatahi) [reply exotic.py:545]: :facepalm:

Comment (tamimfatahi) [reply exotic.py:545]: Done!

Comment (rzellem) [reply exotic.py:545]: lol, to be fair, I am nit-picking :P

**Pull request merged**